internal/util/util.go

```diff
@@ -175,3 +177,14 @@ func Pluralize(word string, count int) string {
 	}
 	return word + "s"
 }
+
+func IsCoreContract(contractName string) bool {
```

Comment:
Any way we can switch Dependency Manager to use this util too? It also has a `isCoreContract` function.